src/ds/d500/d500-private.h

```diff
@@ -107,12 +107,20 @@ namespace librealsense
             float       fy;             /**< Focal length of the image plane, as a multiple of pixel height */
         };
 
+        // These are the possible values for the calibration table 'distortion_model' field
+        enum class d500_calibration_distortion
```

Comment:
Rename d500_distortion_model.

Reply:
Well, I wanted to not make it be confused with non-calibration-table distortion model. I feel like it's easy to get confused between rs2_distortion & d500_distortion_model.
But OK...

Reply:
I went to change it and realized why it's not called 'model' -- the value describes how to translate the distortion coefficients, not just the model. One of the values is brown-then-fisheye, which is not a model. The end model from that is brown. It just means there're more coefficients following those of brown.

What do you want to call this?